perception/multi_object_tracker/src/data_association/data_association.cpp

```diff
@@ -207,6 +207,13 @@ Eigen::MatrixXd DataAssociation::calcScoreMatrix(
         if (passed_gate) {
           const double min_iou = min_iou_matrix_(tracker_label, measurement_label);
           const double min_union_iou_area = 1e-2;
+          if (
+            measurement_object.shape.type == autoware_auto_perception_msgs::msg::Shape::POLYGON &&
+            area == 0 && measurement_object.shape.footprint.points.size() == 1) {
+            measurement_object.shape.footprint.points.front().x = 0.3;
+            measurement_object.shape.footprint.points.front().y = 0.3;
+            measurement_object.shape.footprint.points.front().z = 0.3;
+          }
```

Comment:
To set it to 0.3 would embed an unintended bug.In fact.
It sometimes results in strange behavior when there are actually objects with smaller values.
I think a better way to handle this issue is to have the getIntersectionArea function return 0 if the polygon does not have 3 or more points in it.